frontend/package.json

```diff
@@ -19,6 +19,7 @@
     "date-fns": "^2.29.3",
     "draft-js": "^0.11.7",
     "draftjs-to-html": "^0.9.1",
+    "fast-equals": "^5.0.1",
```

Comment:
Wordt deze nieuwe dependency ergens effectief gebruikt?
If not, delete changes in package(-lock).json

Reply:
Ik ga ervan uit van wel, want de CI testing faalde op het ontbreken van deze library.